### PR TITLE
Refresh refactoring

### PR DESCRIPTION
**Pull request by lauren-li**
## Proposed changes

This PR takes the three `refreshAll` functions for data sets, USS, and jobs from their `actions.ts` files (`refreshAll`/`refreshAllUSS`/`refreshAllJobs`) and combines them into a single `refreshAll` function for simplicity, as the previous code was identical between all 3 trees. 

This PR also removes some unused duplicate functions, and adds comments to clarify the purpose of various functions whose names include `refresh`.

- Current unit and system tests were updated, but no new tests were added as this PR does not introduce any new logic.
- Thanks to @JillieBeanSim for doing the heavy lifting for this PR! I only added the comments.

## Release Notes

Milestone: 1.12

Changelog: (Internal code cleanup; nothing user-facing)

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [x] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Review by katelynienaber — COMMENTED** [review]
The code looks good to me...I only briefly tested the refresh functionality. Is it true that this PR is not meant to fix any specific refresh issues? Ie. the point here is to clean up the code, and refresh in this PR should work the same way as in master...

If so then I can approve it as it works no worse than before XD

**Comment by lauren-li** [comment]
> The code looks good to me...I only briefly tested the refresh functionality. Is it true that this PR is not meant to fix any specific refresh issues? Ie. the point here is to clean up the code, and refresh in this PR should work the same way as in master...
> 
> If so then I can approve it as it works no worse than before XD

Yes, this is just a code cleanup PR. It does not change any of the refresh logic; it just removes duplicate code for it. Thanks @katelynienaber for looking at this PR!

**Review by katelynienaber — APPROVED** [review]
LGTM :)

File: packages/zowe-explorer/src/mvs/mvsNodeActions.ts [hunk 1]
```diff
@@ -1,82 +0,0 @@
...
-            if (favNode) {
-                datasetProvider.refreshElement(favNode);
-            }
-        }
-    } else {
-        vscode.window.showInformationMessage(localize("enterPattern.pattern", "No selection made."));
-    }
-}
-
-export function getDatasetLabel(node: ZoweDatasetNode) {
```
Comment (crawr) [inline packages/zowe-explorer/src/mvs/mvsNodeActions.ts:63]: Was this just a remnant from a previous refactoring? I could not find it used anywhere so it seems we're safe to remove it

**Review by crawr — APPROVED** [review]
Thanks for doing the cleanup. I also like the comments whether the refresh is UI only or not

Didn't find any regressions so it looks good to me 👍

File: packages/zowe-explorer/src/mvs/mvsNodeActions.ts (hunk 1 above)
Comment (lauren-li) [reply packages/zowe-explorer/src/mvs/mvsNodeActions.ts:63]: @crawr I'm not quite sure, but like you, I also couldn't find any references to the functions in `mvsNodeActions.ts` outside of the file itself. They seem to be duplicates of functions we have (and are actively using) elsewhere in the code.

**Pull request merged**